release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2478,6 +2478,196 @@ index a90ab14ff..f7f6f1a71 100644
++
+[source,terminal]
+----
+$ popd
+----
+
+. Generate the ISO file from the `iso-grub-cfg` directory:
++
+[source,terminal]
+----
+$ mkisofs -JR -graft-points -o rhcos-carrier-timeout.iso iso-grub-cfg
+----
+
+. Push the updated ISO image to the server that is accessible by the hub cluster.
+
+. On the hub cluster, update the `osImages` entry in the `AgentServiceConfig` CR for the installing release to point to the updated ISO image:
++
+[source,terminal]
+----
+$ oc edit AgentServiceConfig agent
+----
+
+. Update the `url` field to point to the URL of the updated ISO image.
+
++
+(link:https://issues.redhat.com/browse/OCPBUGSM-46336[*OCPBUGSM-46336*])
+
+* A kernel failure occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+spec:
+  additionalKernelArgs:
+  - rcutree.kthread_prio=11
+----
++
+(link:https://issues.redhat.com/browse/RHELPLAN-123262[*RHELPLAN-123262*])
+
+* During ZTP cluster deployments, bare-metal host image provisioning might fail with an error referencing the HTTP 412 error code:
++
+[source,terminal]
+----
+Deploy step deploy.deploy failed with HTTPError: HTTP PATCH https://10.16.230.34/redfish/v1/Managers/1/VirtualMedia/EXT1 returned code 412. Base.1.8.GeneralError: A general error has occurred. See ExtendedInfo for more information. Extended information: [
+
+{'MessageSeverity': 'Critical', 'MessageArgs': [], 'MessageId': 'Base.1.8.PreconditionFailed', 'Resolution': 'Try the operation again using the appropriate ETag.', '@odata.type': '#Message.v1_1_0.Message', 'Message': 'The ETag supplied did not match the ETag required to change this resource.'}
+----
+
++
+This issue might affect various server models, in particular Gen8 HP machines and Gen9 HP machines running older firmware. For Gen9 HP machines, upgrading to the latest iLO firmware might resolve the issue. For Gen8 HP and others machines, there is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-1246[*OCPBUGS-1246*])
+
+* On certain Lenovo models, for example the SE450, bare-metal host image provisioning during ZTP cluster deployments might fail with the HTTP 400 status code and a `PropertyNotWriteable` error:
++
+[source,terminal]
+----
+HTTP response for PATCH https://192.168.26.178/redfish/v1/Systems/1/Pending: status code: 400, error: Base.1.8.GeneralError: A general error has occurred. See ExtendedInfo for more information., extended: [{'MessageArgs': ['BootSourceOverrideEnabled'], 'Resolution': 'Remove the property from the request body and resubmit the request if the operation failed.', MessageId': 'Base.1.8.PropertyNotWritable', 'Message': 'The property BootSourceOverrideEnabled is a read only property and cannot be assigned a value.', '@odata.type': '#Message.v1_1_0.Message', 'MessageSeverity': 'Warning'}
```

Comment:
First `[` does not back a closing bracket

Reply:
Added a closing bracket here as well